light-client/src/evidence.rs

```diff
@@ -1,19 +1,18 @@
 //! Fork evidence data structures and interfaces.
 
-use crate::{components::io::IoError, types::PeerId};
+use futures::future::FutureExt as _;
 
 use tendermint::abci::transaction::Hash;
-
-use contracts::contract_trait;
-
 pub use tendermint::evidence::Evidence;
 
+use crate::{components::io::IoError, types::PeerId, utils::time::timeout};
```

Comment:
Can we feature-guard this import so that it doesn't trigger an unused warning when building for wasm?